liquidprompt

```diff
@@ -1289,6 +1291,28 @@ _lp_dirstack_color() {
     lp_dirstack_color="${LP_COLOR_DIRSTACK}${LP_MARK_DIRSTACK}${lp_dirstack}${NO_COL}"
 }
 
+_lp_aws_profile() {
+    (( LP_ENABLE_AWS_PROFILE )) || return 2
+
+    local ret
+
+    if [[ -n "${AWS_PROFILE-}" ]]; then
+        __lp_escape "${AWS_PROFILE##*/}"
+        lp_aws_profile=$ret
+    elif [[ -n "${AWS_DEFAULT_PROFILE-}" ]]; then
+        __lp_escape "${AWS_PROFILE##*/}"
+        lp_aws_profile=$ret
```

Comment:
This whole block could be simplified a bit to reduce duplication:
```suggestion
    local aws_profile="${AWS_PROFILE-${AWS_DEFAULT_PROFILE-}}"
    if [[ -n $aws_profile ]]; then
        __lp_escape "${aws_profile##*/}"
        lp_aws_profile=$ret
```

Reply:
Good call, done.